_pages/MitoJohnDria.yml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: MitoJohnDria
+pageUrl: MitoJohnDria.github.io
+timestamp: 2017-02-24
+---
```

Comment:
@MitoJohnDria just add a line break at the end of line 5 and you should be able to merge!